src/overlays/actors/ovl_En_Weather_Tag/z_en_weather_tag.h

```diff
@@ -19,8 +19,6 @@ typedef struct EnWeatherTag {
     /* 0x158 */ u16 unk158; // increment/decrement amount for 154
 } EnWeatherTag; // size = 0x15C
 
-extern const ActorInit En_Weather_Tag_InitVars;
-
 #define WEATHER_TAG_TYPE(x) (x->actor.params & 0xF)
```

Comment:
And. Indeed, these macros are totally inconsistent with the rest of the codebase.